server/candlepin.spec.tmpl

```diff
@@ -40,7 +40,11 @@ BuildRequires: /usr/bin/pathfix.py
 
 Requires: java >= 1:1.8.0
 Requires: wget
+%if 0%{?rhel} == 8
+Requires: pki-servlet-container
```

Comment:
I wonder if this should be a bit more generic and require `bundled(tomcat)`

Reply:
I had the same question but according to https://fedoraproject.org/wiki/Bundled_Libraries?rd=Packaging:Bundled_Libraries , at least in Fedora, the virtual bundled provides are not intended for use in requires statements, and are mainly for making package maintenance easier.